pkg/block/block.go

```diff
@@ -222,3 +222,21 @@ func IsBlockDir(path string) (id ulid.ULID, ok bool) {
 	id, err := ulid.Parse(filepath.Base(path))
 	return id, err == nil
 }
+
+// Returns list of segment files for given block. Paths are relative to the chunks directory.
```

Comment:
For goDoc/go.dev to work properly: 

```suggestion
// GetSegmentFiles returns list of segment files for given block. Paths are relative to the chunks directory.
```